packages/runtime/container-runtime/.eslintrc.cjs

```diff
@@ -81,15 +82,12 @@ module.exports = {
 			files: ["*.spec.ts", "src/test/**"],
 			rules: {
 				// TODO: remove these overrides and fix violations
-				"@typescript-eslint/explicit-function-return-type": "off",
+				"@typescript-eslint/explicit-function-return-type": "warn",
+				"unicorn/consistent-function-scoping": "warn",
 				"unicorn/error-message": "warn",
 
 				// Test files are run in node only so additional node libraries can be used.
 				"import/no-nodejs-modules": ["error", { allow: ["node:assert", "node:crypto"] }],
-
-				// TODO:AB#3027: This rule is disabled for tests in the `recommended` base config.
```

Comment:
This was apparently incorrect. This disable remains, though set to a warning. Cleanup can be done later with low urgency.